src/lib/GifEncoder.ts

```diff
@@ -335,7 +335,7 @@ export class GifEncoder {
 	 * received so that timing data can be inserted. Calling {@link GifEncoder.finish} will flush all frames.
 	 * @param imageData The image data to add into the next frame.
 	 */
-	public addFrame(imageData: CanvasRenderingContext2D | Uint8ClampedArray) {
+	public addFrame(imageData: ResolvableImage | Uint8ClampedArray) {
```

Comment:
Is this a `fix` or `feature`? Regardless, this PR is not a `chore` because there is a code change that will affect end-users. The PR title doesn't encompass the changes made.

Reply:
I have no idea, it's just a narrowed version of `CanvasRenderingContext2D` with the only method we use from it.